src/components/Placeholder/index.tsx

```diff
@@ -35,11 +35,11 @@ const StyledDiv = styled.div`
   opacity: 0.8;
   background: linear-gradient(
     90deg,
-    rgba(255, 255, 255, 0),
-    rgba(255, 255, 255, 0.3),
-    rgba(255, 255, 255, 0.4),
-    rgba(255, 255, 255, 0.3),
-    rgba(255, 255, 255, 0)
+    ${({ theme }) => theme.colors.neutral.backgroundWeak}00,
```

Comment:
what ???

Reply:
@philibea we wanted tokens from theme + opacity, in hexa you can simply add values at the end it to change opacity which we did here

ref: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4